staging/src/k8s.io/apiserver/pkg/endpoints/handlers/delete.go

```diff
@@ -104,7 +104,8 @@ func DeleteResource(r rest.GracefulDeleter, allowsOptions bool, scope *RequestSc
 				trace.Step("Decoded delete options")
 
 				ae := request.AuditEventFrom(ctx)
-				audit.LogRequestObject(ae, obj, scope.Resource, scope.Subresource, scope.Serializer)
+				objGVR := schema.GroupVersionResource{Group: scope.Kind.Group, Version: scope.Kind.Version, Resource: scope.Kind.Kind}
```

Comment:
similar comment here, `Resource: scope.Kind.Kind` is not correct, need to pass in the distinct coordinates of the request (used to set ObjectRef) and the request object group-version (used to encode the object)

Reply:
thanks,
done